app/src/components/grids/cardsGrid.tsx

```diff
@@ -16,44 +16,49 @@ interface CardsGridProps {
   records: CollectionDataType[] | ItemDataType[];
 }
 
-export const CardsGrid = ({ records }: CardsGridProps) => {
-  const { isLargerThanLargeTablet } = useBreakpoints();
-  const isCollections = isCollectionType(records);
-  const cardRef = useRef<HTMLDivElement>(null);
-  const tooltipOffset = useTooltipOffset(cardRef);
-  const imageHeight = useCardImageHeight(cardRef);
+export const CardsGrid = forwardRef<HTMLDivElement, CardsGridProps>(
+  ({ records }, ref?) => {
```

Comment:
> How do we feel about the ref going to the first card when the page changes?

Think it makes sense. See what Claire says but I don't see a problem with this? Maybe also ask Lana as she can fill in for Lily atm. @EdwinGuzman what do you think?

Reply:
see [comment](https://github.com/NYPL/digital-collections/pull/216#issuecomment-2450181288)